thunder/tests/opinfos.py

```diff
@@ -7238,6 +7238,58 @@ def nll_loss_error_generator(op, device, dtype=torch.float32, **kwargs):
 nn_ops.append(nll_loss_opinfo)
 
 
+def mse_loss_sample_generator(op, device, dtype, requires_grad, **kwards):
+    make = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    # input_shape, target_shape
+    shapes = (
+        ((2, 16), (2, 16)),
```

Comment:
What about some examples where the shape of the input and the shape of the target differ?

Reply:
Ah, sure! From the warning in PyTorch shown below, I thought having different shaped input and target is an undesirable usage of `mse_loss` so didn't add a test case for that.
```
            "This will likely lead to incorrect results due to broadcasting."
            "Please ensure they have the same size."
```
However, I do think it is something good to have :)